subjects/collections/README.md

```diff
@@ -37,7 +36,7 @@ map.set(3, 'c')
 map.set(4, 'd')
 
 arrToSet(arr) // -> Set { 1, 2, 3 }
-arrToStr(arr) // -> '1213'
+arrToStr(arr) // -> '123'
```

Comment:
This was correct like it was, now the output is wrong. The array is arr = [1, 2, 1, 3]